build/vega-lite-schema.json

```diff
@@ -21928,6 +21928,12 @@
             {
               "$ref": "#/definitions/ColorScheme"
             },
+            {
```

Comment:
Confirmed that this matches what we see in the underlying `Vega` types for RangeScheme 👍 

https://github.com/vega/vega/blob/b8353d57e4842c88b81fbcfbfd7ccf48b867ae2b/packages/vega-typings/types/spec/scale.d.ts#L21